### PR TITLE
Private deployments sprint wrap-up docs

### PR DESCRIPTION
**Pull request by ccooper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,29 +86,48 @@ Any issues that are purely process-based (e.g. scrum documentaion) should be fil
...
 As we near the end of the current sprint, the milestone for the next sprint is often obvious based on organizational needs or follow-on work that builds on the current sprint. If there are multiple milestones possible, they should all be added to the roadmap. This helps keep customers and external parties informed.
 
-If multiple future sprints are ready and there are no other factors to aid decision, the _Scrum master_ will decide.
+If multiple future sprints are ready and there are no other factors to aid decision, the _Scrum master_ will decide which sprint will come next.
 
 ### Workspace configuration
 We have the following pipelines (columns) configured in Zenhub:
 * **Icebox**: issues we will never work on. These may be valid, but are not a priority given finite time and resources.
-* **New Issues**: newly-filed issues. They will remain here until triaged and labelled to go into either the Product Backlog, Next Sprint, Current Sprint Backlog, In Progress, or Icebox pipeline.
+* **New Issues**: newly-filed issues. They will remain here until triaged and labeled to go into either the Product Backlog, Next Sprint, Current Sprint Backlog, In Progress, or Icebox pipeline.
```
Comment (djmitche) [inline README.md:128]: I think we're actually using these at the reverse of what's described here -- new issues are automatically put in icebox, and then triaged into either "new issues" (that we are not planning on working on) and "product backlog" (that will eventually be worked on).

File: sprints/20200928-private-deployments/sprint-retrospective.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Sprint retrospective - Sprint 6: Private Taskcluster deployments, September 28 - October 23, 2020
+
+## What worked
+* Better appreciation for worker space
+  * Like the last sprint, this "extra" epic validated that all epics in a sprint don't need to address the same initiative. Both [@djmitche](https://github.com/djmitche) and [@imbstack](https://github.com/imbstack)
```
Comment (djmitche) [inline sprints/20200928-private-deployments/sprint-retrospective.md]: both of us .. ?

**Review by djmitche — APPROVED** [review]

File: sprints/20200928-private-deployments/sprint-retrospective.md (hunk 2 above)
Comment (ccooper) [reply sprints/20200928-private-deployments/sprint-retrospective.md]: Oops, stopped mid-thought. Fixed.

File: README.md (hunk 1 above)
Comment (ccooper) [reply README.md:128]: By default, they do go to the Icebox if the person filing doesn't explicitly change the pipeline. I think that's an artifact of how the board was initially setup.

I would like people to try to remember to change the pipeline, but I'm also trying to get better about triaging issues out of there more regularly.

Comment (djmitche) [reply README.md:128]: I think that's what I mean: we triage things *out of* icebox, where the description above says that we triage things *into* icebox.

**Pull request merged**